plugins/org.jkiss.dbeaver.model.rcp/src/org/jkiss/dbeaver/model/navigator/DBNResource.java

```diff
@@ -420,6 +420,13 @@ public <T> T getAdapter(Class<T> adapter) {
             if (adapter == Path.class) {
                 IPath location = resource.getLocation();
                 return location == null ? null : adapter.cast(location.toPath());
+            } else if (adapter == InputStream.class && resource instanceof IFile file) {
+                try {
+                    return adapter.cast(file.getContents());
+                } catch (CoreException e) {
+                    log.error("Error get file content");
+                    throw new RuntimeException(e);
```

Comment:
You shouldn't really throw unchecked exceptions. Check the contract of `IAdaptable`. Just log and return `null`, or fall back to the super implementation.